src/com/facebook/buck/android/AndroidBinaryGraphEnhancer.java

```diff
@@ -597,7 +600,8 @@ PreDexMerge createPreDexMergeRule(
         allPreDexDeps,
         dexForUberRDotJava,
         dxExecutorService,
-        xzCompressionLevel);
+        xzCompressionLevel,
+        dxConfig.getDxMaxHeapSize());
```

Comment:
Unclear whether this is necessary, because merging is much less memory intensive, and we usually run it in-process, but okay.